### PR TITLE
Add Nelo logo

### PR DESCRIPTION
**Pull request by shebson**
Nelo is a BNPL provider in Mexico. This icon will be used by Nelo's Shopify Payment App. Nelo's brand guidelines are available [here](https://docs.nelo.mx/docs/logos-and-brand).

** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by shebson** [comment]
For context, this is how Nelo's icon looks alongside other BNPL, PSP and card network icons in this repo:

web-logos

File: app/assets/images/payment_icons/nelo.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-nelo">
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/nelo.svg:1]: Can we put the code in 1 line just like the rest of brand icons?

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

**Review by tauthomas01 — COMMENTED** [review]
Hi, this looks good. I just left one small comment

File: app/assets/images/payment_icons/nelo.svg (hunk 1 above)
Comment (shebson) [reply app/assets/images/payment_icons/nelo.svg]: Absolutely! Just updated the PR to address this.

**Comment by shebson** [comment]
Thank you for the speedy review @tauthomas01! I have pushed up a commit to collapse this to a single line like the other SVGs.

**Review by larouxn — APPROVED** [review]

**Review by tauthomas01 — APPROVED** [review]

**Comment by tauthomas01** [comment]
Looks great! Can you rebase your branch off `master` and fix the conflicts?

**Comment by shebson** [comment]
@tauthomas01 I've rebased on master and this should be good to go :-)

**Comment by larouxn** [comment]
👋 Just a heads up that we are hoping to have this live across Shopify within the month. Currently waiting on some other icons so we can batch the update. Thank you. 🙇‍♂️

**Comment by larouxn** [comment]
👋  Ran into a few release pipeline issues. Hoping to have this live by end of next week, potentially sooner.

**Pull request merged**